README.md

```diff
@@ -1,7 +1,8 @@
 # krill-sync
 
-A tool to synchronize an RRDP and/or Rsync server with a "hidden" remote 
-[RFC 8182](https://tools.ietf.org/html/rfc8182) RRDP publication point.
+This tool intended to facilitate a redundant setup where multiple public https
```

Comment:
s/tool intended/tool is intended/. Additionally, I find this text a bit hard to parse and it leaves me confused. Why would someone use krill-sync to facilitate a redundant setup and not some other more established means? The whole point of krill-sync is that it is RRDP aware and can thus replicate repository content without breaking its logical consistency from the perspective of RP clients.

Reply:
improved, I hope